src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo__link">
+        <a href="/">
+          <img
+            src="images/logo.png"
+            alt="Company logo"
+          />
+        </a>
```

Comment:
The class name 'logo__link' suggests that it is an element of a 'logo' block, but such block does not exist. Consider renaming to 'header__logo' if it is part of the header block.